[BUG] Krkn run failing with an assertion error ApiClient.call_api() got an unexpected keyword argument 'response_type'

# Bug Description

Used Krkn Release : 5.1.0v

AssertionError: Krkn run failed for osd_only grouped application outage. Found 1 error(s) in output file /home/jenkins/workspace/qe-deploy-ocs-cluster/ocs-ci/data/krkn_scenarios/qyywgcbv/output.log. First error: 2026-07-02 02:12:45,343 [ERROR] failed to create token for SA: prometheus-k8s on namespace: openshift-monitoring with error: ApiClient.call_api() got an unexpected keyword argument 'response_type'

[application_outage.yaml](https://github.com/user-attachments/files/29585651/application_outage.yaml)
[krkn_global_config.yaml](https://github.com/user-attachments/files/29585653/krkn_global_config.yaml)
[output.log](https://github.com/user-attachments/files/29585652/output.log)

## **Describe the bug**

A clear and concise description of what the bug is.

## **To Reproduce**

Any specific steps used to reproduce the behavior

### Scenario File
Scenario file(s) that were specified in your config file (can be starred (*) with confidential information )
```yaml
<config>

```

### Config File 
Config file you used when error was seen (the default used is config/config.yaml)

```yaml
<config>

```

## **Expected behavior**

A clear and concise description of what you expected to happen.

## **Krkn Output**

Krkn output to help show your problem 

## **Additional context**

Add any other context about the problem

--- a/requirements.txt
+++ b/requirements.txt
@@ -19,7 +19,7 @@ jaraco-context>=6.1.0  # Fixes GHSA-58pv-8j8x-9vj2
 cbor2<5.7.0  # Pinned by arcaflow-plugin-sdk
 lxml==6.1.0
 kubernetes>=35.0.0,<36.0.0
-krkn-lib==6.1.0
+krkn-lib==6.1.2
 numpy==1.26.4
 pandas==2.2.0
 openshift-client==1.0.21

